mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java

```diff
@@ -449,6 +449,7 @@ public void processConnection() throws Exception {
               .protocol(protocol)
               .listener(framedSocketListener)
               .build();
+          framedConnection.start();
```

Comment:
We weren’t sending a connection preface. Though this didn’t matter for SPDY/3, it’s non-compliant for HTTP/2. We didn’t catch it because OkHttp doesn’t enforce this.